site_conditions/USGS_GLWBSRV_Conditions.yaml

```diff
@@ -0,0 +1,112 @@
+ProtocolId: USGS_GLWBSRV_Conditions
+ProtocolName: 'USGS_GLWBSRV Aerial Survey Conditions'
+ProtocolDescriptionURL: ''
+DoubleObserverInd: 0
+IncludesDetectionAidInd: 0
+ProtocolTypeCd: SiteConditions
+CategorySets:
+    -
+        ConditionCategorySetId: Weather
+        MultiplesAllowed: 0
+        ConditionCategorySetDescription: 'weather data'
+        DisplayOrder: 1
+        ConditionCategorySetRules: null
+        Locales:
+            en:
+                ConditionCategorySetLabel: Weather
+        Categories:
+            -
+                ConditionCategoryId: WindDir
+                ConditionCategoryDescription: 'General wind direction in degrees'
+                ConditionCategoryDataTypeCd: integer
+                DefaultValue: null
+                ConditionCategoryRules: 'regex'
+                ConditionCategoryRuleData: '/^[0-355]$/'
+                DisplayOrder: 1
+                Help: null
+                Locales:
+                    en:
+                        ConditionCategoryLabelBefore: 'Wind direction (deg)'
+                        ConditionCategoryLabelAfter: null
```

Comment:
LabelAfter should be the units of measurement, (in this case "deg") and you can remove that from LabelBefore